lib/validate.js

```diff
@@ -52,6 +52,13 @@ function contextAppend(options, s) {
     options.context.push((options.context[options.context.length - 1] + '/' + s).split('//').join('/'));
 }
 
+const lintObject = (objectName, object, options) => {
+    const results = options.linter(objectName, object, options);
+
+    // Update results
```

Comment:
This method is mostly there to avoid needing to refactor validate.js to look for the new return value. In the future I'd like to continue this work, but I'm waiting for the dust to settle on openapi-kit before I worry about any of that. Let's accept this slight oddity and move on.